app/routes/payments/$paymentId.tsx

```diff
@@ -26,25 +49,399 @@ export function ErrorBoundary({ error }: { error: Error }) {
+            theme.palette.blue.light
+          )}
+          {boxWithCopyToClipboard(
+            t('pages.payment.reference'),
+            details.id,
+            theme.palette.violet.light
+          )}
+        </Box>
+        <DividerWithSpace />
+        <Box
+          sx={{
+            display: 'flex',
+            width: '100%',
+            justifyContent: 'space-around',
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              width: '45%',
+              flexDirection: 'column',
+              justifyContent: 'start',
+            }}
+          >
+            {dataItem(
+              t('pages.payment.type'),
+              chipsWithColor(
+                'Pay-in',
+                theme.palette.green.light,
+                theme.palette.green.darker
+              )
+            )}
+            {dataItem(
+              t('pages.payment.processor'),
+              <Box sx={{ m: '10px' }}>{details.provider}</Box>
+            )}
+            {dataItem(
+              t('pages.payment.status'),
+              chipsWithColor(
+                details.status,
+                theme.palette.green.light,
+                theme.palette.green.darker
+              )
+            )}
+          </Box>
+
+          <Box
+            sx={{
+              display: 'flex',
+              width: '45%',
+              flexDirection: 'column',
+            }}
+          >
+            {dataItem(
+              t('pages.payment.netValue'),
+              <>{details.initialAmount}</>
+            )}
+            {dataItem(
+              t('pages.payment.initialAmount'),
+              <>{details.initialAmount}</>
+            )}
+          </Box>
+        </Box>
+        <DividerWithSpace />
+        <Typography variant="headline">
+          {t('pages.payment.eventJournal.title')}
+        </Typography>
+        {details.adjustments.map(
+          (adjustments: AdjustmentsItem, index: number) => (
+            <div key={index}>
+              {eventsJournalItem(
+                t('pages.payment.eventJournal.statusChange', {
+                  value1: details.initialAmount,
+                  value2: adjustments.amount,
+                }),
+                adjustments.date
+              )}
+              {eventsJournalItem(
+                t('pages.payment.eventJournal.netValueChange', {
+                  value1: details.status,
+                  value2: adjustments.status,
+                }),
+                adjustments.date
+              )}
+            </div>
+          )
+        )}
+        <DividerWithSpace />
+        <Typography variant="headline" sx={{ pb: '10px' }}>
+          {t('pages.payment.reconciliation.title')}
+        </Typography>
+        <Typography
+          variant="footNote"
+          sx={{ color: ({ palette }) => palette.grey[600] }}
+        >
+          {t('pages.payment.reconciliation.subTitle')}
+        </Typography>
+
+        <Table
```

Comment:
you need to use Table not from @numaryhq/storybook but from the wrapper to avoid having to handle labels, pagination...

the wrapper is in src/components/wrapper/Table and it uses the Table from @numaryhq/storybook